.travis.yml

```diff
@@ -42,3 +42,14 @@ addons:
 script:
   - if [ "$TRAVIS_OS_NAME" == "linux" ]; then sudo docker exec mantella /bin/bash -c "./.develop.sh --test --install"; fi
   - if [ "$TRAVIS_OS_NAME" == "osx" ]; then ./.develop.sh --test --install /usr/local/include; fi
+
+before_deploy:
+  -  'COMMIT_MESSAGE="Auto doc : $(git log --pretty="%s - %an" -1)"'
+  
+deploy:
+  skip_cleanup: true
+  provider: script
+  on:
+    condition: "$CC = gcc && $TRAVIS_OS_NAME = linux"
+    branch: master 
+  script: sudo docker exec mantella /bin/bash -c "./.develop.sh --doc"; openssl aes-256-cbc -K $encrypted_f98812e554e5_key -iv $encrypted_f98812e554e5_iv -in ./doc/docsecret.enc -out ./doc/docsecret -d; chmod 400 ./doc/docsecret; ssh-agent bash -c 'ssh-add ./doc/docsecret; git clone --quiet git@github.com:Mantella/Mantella.info.git'; cp -r ./doc/_html/* ./Mantella.info/; cd Mantella.info/; git add -A; git commit -m "${COMMIT_MESSAGE}"; ssh-agent bash -c 'ssh-add ../doc/docsecret; git push --quiet origin master';
```

Comment:
git add ... gut commit -m can't be replaced by git commit -a -m because 
```
-a
--all
Tell the command to automatically stage files that have been modified and deleted, but new files you have not told Git about are not affected.
```
 we have many of new files